www/assets/js/components/Facet.tsx

```diff
@@ -71,11 +81,4 @@ function SearchFacet(props: Props) {
   )
 }
 
-const propsAreEqual = (_prevProps: Props, nextProps: Props) => {
```

Comment:
We can't use this anymore since the aggregation is no longer nested in the bucket key - but the facets don't appear and disappear so I think the issue has been fixed elsewhere